pkg/reconciler/inmemorychannel/controller/inmemorychannel_test.go

```diff
@@ -83,6 +84,11 @@ var (
 		URL:  apis.HTTP("test-imc-kn-channel.test-namespace.svc.cluster.local"),
 	}
 
+	channelAudience = auth.GetAudience(v1.SchemeGroupVersion.WithKind("InMemoryChannel"), metav1.ObjectMeta{
+		Name:      imcName,
+		Namespace: testNS,
+	})
```

Comment:
Ideally, in tests, we use non production code so that we detect breaking changes to the `GetAudience` function, in this case, I'd hardcode the expected audience

Reply:
Just wanted to make sure, we don't have to update the unit tests, when we change the implementation of GetAudience() (for what we have a unit test too). But I see...
Updated it...